libs/fgdbg/src/DebugServer.cpp

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright (C) 2022 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "fgdbg/DebugServer.h"
+#include <CivetServer.h>
+#include <utils/Log.h>
+#include <tsl/robin_set.h>
+
+using namespace utils;
+namespace filament::fgdbg {
+
+using namespace utils;
```

Comment:
You have this `using` line twice

Reply:
I **really** want to use it
haha good catch thanks